shared/chat/conversation/messages/attachment.desktop.js

```diff
@@ -0,0 +1,63 @@
+// @flow
+import React, {Component} from 'react'
+import {Avatar, Box, Icon, Text} from '../../../common-adapters'
+import {globalStyles, globalMargins, globalColors} from '../../../styles'
+import * as Constants from '../../../constants/chat'
+
+import type {Props} from './attachment'
+
+const _marginColor = (followState) => ({
```

Comment:
this is copy/pasted from messages/text.desktop this should be cleaned when we work on the attachment ui. Outside of the scope of this PR